.github/workflows/nightlyAndMainImage.yml

```diff
@@ -50,11 +47,17 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DOCKER_IMAGE_NAME: newrelic/nri-kubernetes
-      DOCKER_IMAGE_TAG: nightly
       DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm" # Must be consistent with the matrix from the job above
     steps:
       - uses: actions/checkout@v2
-
+      - name: Setting Docker image tag
+        run: |
+          if [[ "${{ github.ref_name }}" == "main" ]]
```

Comment:
What is `ref_name` and how is it used to distinguish cron/merge?

Reply:
somehow I assumed without even thinking about that, that the ref_name was going to be different. The assomption should be completely wrong. 🙇 

I will check that

Reply:
In other workflows I've used `github.event`, as will have an object with the name of the event type.

For example, `${{ if github.event.schedule }}` should evaluate to true if the event runs on schedule:

https://github.com/newrelic/nri-kubernetes/blob/a0ed2ac4810af825bd8b9f69f3fdd1692ad7c697/.github/workflows/security.yaml#L23

Reply:
I'm not sure how that could be put in a bash `if`... but I guess we can always have two steps setting the variable, and `skip` based on that :hack:

Reply:
I added a default and a set that can be skipped

Reply:
@roobre It feels like a bit an hack, let me know what you think